src/map/battle.c

```diff
@@ -4843,7 +4849,9 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 			case MO_EXTREMITYFIST:
 			case GS_PIERCINGSHOT:
 			case AM_ACIDTERROR:
+#ifndef RENEWAL
```

Comment:
I think it was actually an oversight from me, since part of the change is to make this skill "not distinct" (in other words, use default formula instead of special one).

I remember I did the same thing in the other case below but noticed it before pushing. so simply removing this case will be enough